### ISSUE TITLE
Filtering message search

### ISSUE THREAD

**Issue by CATER-COM** [nextcloud/mail#2957]
### Feature Request

Adding ability to specifiy the field you want to search

#### Summary

Actualy, yhe seach feature seem to seach a term in the whole content. It could be nice to specifiy if the searched term need to be in subject or address or body. I,think it should be faster as the field of the search is restricted.

When you click in the searh box, it could provide de dropdown list suggestion to search "all", "from", "to", "subject" and "body".

Sorry if it's a duplicate but I don't found issue on this topic.

Good evening.

**Comment by ChristophWurst** [comment]
This is actually supported already. But since the search is so disconnected there is no space to add explanatory text. In any case, you may try searches like `to:user@domain.tld` or `subject:test`

**Comment by ChristophWurst** [comment]
Btw @jancborchardt something to consider for the new search :)

**Comment by ChristophWurst** [comment]
We could also add these power user searches to the shortcuts page. What do you think?

**Comment by jancborchardt** [comment]
> We could also add these power user searches to the shortcuts page. What do you think?

Yep, that’s a good inbetween fix, especially as shortcuts are exactly what they are. :)

**Comment by CATER-COM** [comment]
> This is actually supported already. But since the search is so disconnected there is no space to add explanatory text. In any case, you may try searches like to:user@domain.tld or subject:test

@ChristophWurst  Nice good to know. Is there a specific synthax with alias/joker/subsitute string like * ou % ? Because searching the full address or the full subject, in not easy. You usually don't know the excact terms of the searched string. 

@jancborchardt what is this shortcuts page ? Is it something already implemented or something coming with next realease ?

**Comment by admorgan** [comment]
@jancborchardt if you open the settings there is a button to show "Keyboard Shortcuts"
